src/pages/ProductDetail/index.js

```diff
@@ -1,9 +1,13 @@
 import React, { Component } from 'react'
```

Comment:
Right now there's a bug in the application - try opening Product Detail page, refresh the page, and then add the product to cart and navigate to the cart. Since we never opened Product List page, `products` property in the global state never got populated. So you might want to add a single product from Product Detail page to `products` array when you navigate to Product Detail page

Reply:
i am not sure how to do that - i tried to use the fetchProduct method i am already using and create products property in the global state, but it seems it has a bit different structure (the price is missing) and the app breaks when i return to homepage (sent in git). should i maybe just load all the products on detail, filter the currently viewed from them for display and ditch the local state product in product's detail all together?